shared/chat/inbox/row/index.tsx

```diff
@@ -17,58 +14,40 @@ const makeRow = (
   closeOpenedRow: () => void
 ) => {
   if (item.type === 'bigTeamsLabel') {
-    return (
-      <Box style={_bigTeamLabelStyle}>
-        <BigTeamsLabel />
-      </Box>
-    )
+    return <BigTeamsLabel />
   }
   switch (item.type) {
     case 'bigHeader':
-      return (
-        <C.ChatProvider id={C.Chat.dummyConversationIDKey}>
-          <BigTeamHeader teamname={item.teamname} teamID={item.teamID} navKey={navKey} />
-        </C.ChatProvider>
-      )
+      return <BigTeamHeader teamname={item.teamname} teamID={item.teamID} navKey={navKey} />
     case 'big':
       return (
-        <C.ChatProvider id={item.conversationIDKey}>
```

Comment:
since this is a function and not a component its not memoized so we always thrash the provider since there's a new child each time, instead we push this down inside a memo